analyzer.go

```diff
@@ -68,12 +68,13 @@ var lintersOptions = map[string]map[string]argumentConstructor{
 func (a *Analyzer) NotifyReviewEvent(ctx context.Context, e *pb.ReviewEvent) (
 	*pb.EventResponse, error) {
 	changes, err := a.DataClient.GetChanges(ctx, &pb.ChangesRequest{
-		Head:            &e.Head,
-		Base:            &e.Base,
-		WantContents:    true,
-		WantLanguage:    true,
-		WantUAST:        false,
-		ExcludeVendored: true,
+		Head:             &e.Head,
+		Base:             &e.Base,
+		WantContents:     true,
+		WantLanguage:     true,
+		WantUAST:         false,
+		ExcludeVendored:  true,
+		IncludeLanguages: []string{"go"},
```

Comment:
Is this case-sensitive? Do we have a list of languages? Such in this case I don't know if it is `go` or `golang` for example.

Reply:
related: https://github.com/src-d/lookout-sdk/issues/62